test/lint-spelling.js

```diff
@@ -206,7 +219,11 @@ windows:${M}\r
       <emu-alg>
         1. [id="step-label"] Foo.
       </emu-alg>
-      Something about step <emu-xref href="#step-label"></emu-xref>.
+      <p>Something about step <emu-xref href="#step-label"></emu-xref>.</p>
+
+      <table><tr><td>Spaces for aligning table cells are acceptable.</td>
+      <td>For example, it is nice for these to line up.             </td>
```

Comment:
I don't really care to add this exception. If we did this, I'd think we'd also want to allow `</li>`, `</dt>`, `<dd>`, etc. So probably either allow any whitespace until `<` or disallow all internal whitespace. Is this exception necessary for ecma262 to pass currently?

Reply:
> Is this exception necessary for ecma262 to pass currently?

Yes, see [discussion above](https://github.com/tc39/ecmarkup/pull/239#discussion_r471938869).

Reply:
Then I think we should add some more of those other reasonable tags to the list or just forbid it entirely.

Reply:
I am happy to add other tags as well where it makes sense to do so, though I disagree that any of `</li>`,  `</dt>,` or `</dd>` would be good candidates: I would not expect to see multiple such tags on a single line, and so would not expect to see internal spaces used for alignment. Do you have other tags you'd like to add?